### DISCUSSION TITLE
Does elsa support BPMN?

### DISCUSSION THREAD

**Discussion by zhouguoqing** [elsa-workflows/elsa-core#639] [Q&A]
Hi Sipke , I am a elsa-workflows's fans. It's a great workflow framework as I know.
So we want to use it as our workflow engine. Now,  I have a question as following,
Does elsa  support BPMN?

**Comment by zbogdanov** [comment]
@zhouguoqing this is a duplicate, take a look here: https://github.com/elsa-workflows/elsa-core/issues/460 or https://github.com/elsa-workflows/elsa-core/issues/39 here
